src/convertToObject.js

```diff
@@ -7,6 +7,26 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  // eslint-disable-next-line no-debugger
+  if (!sourceString) {
+    return {};
+  }
+
+  const styleToObject = {};
+  const partsOfExpresions = sourceString.split(';');
+
+  partsOfExpresions.forEach((part) => {
+    const index = part.indexOf(':');
+
+    const name = part.substring(0, index).trim();
+    const value = part.substring(index + 1).trim();
+
+    if (name && value) {
+      styleToObject[name] = value;
+    }
+  });
```

Comment:
The use of 'forEach' here is appropriate as it does not return a new array and is used to iterate over the array to perform operations. However, consider using more modern iteration methods like 'map' or 'filter' if you need to transform or filter the array and then assign the result to a new variable.